tests/create_entity_parent_test.py

```diff
@@ -0,0 +1,43 @@
+#  Copyright (c) 2021 Mira Geoscience Ltd.
+#
+#  This file is part of geoh5py.
+#
+#  geoh5py is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  geoh5py is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with geoh5py.  If not, see <https://www.gnu.org/licenses/>.
+
+import tempfile
+from pathlib import Path
+
+from geoh5py.groups import ContainerGroup
+from geoh5py.objects import Points
+from geoh5py.workspace import Workspace
+
+
+def test_create_point_data():
+
+    with tempfile.TemporaryDirectory() as tempdir:
+        h5file_path = Path(tempdir) / r"test.geoh5"
```

Comment:
with the fixture, will become
```suggestion
        h5file_path = tmp_path / r"test.geoh5"
```